deployment/service_template_generator.py

```diff
@@ -175,29 +179,80 @@ def _add_service(self, service_name, config):
             container_definition_arguments['Command'] = [config['command']]
 
         cd = ContainerDefinition(**container_definition_arguments)
-        td = TaskDefinition(
-            service_name + "TaskDefinition",
-            Family=service_name + "Family",
-            ContainerDefinitions=[cd]
-        )
+        if launch_type == self.LAUNCH_TYPE_FARGATE:
+            td = TaskDefinition(
+                service_name + "TaskDefinition",
+                Family=service_name + "Family",
+                ContainerDefinitions=[cd],
+                RequiresCompatibilities=['FARGATE'],
+                ExecutionRoleArn='arn:aws:iam::725827686899:role/ecsTaskExecutionRole',
```

Comment:
We should use get-role api to retrieve this ARN instead of coding in this account ID
```python
boto3.resource('iam').Role('ecsTaskExecutionRole').arn
```

Reply:
Fixed in c7fbadd4